### ISSUE TITLE
PRs not opened on schedule for some repos

### ISSUE THREAD

**Issue by piotr-s-brainhub** [renovatebot/renovate#3303]
**What Renovate type are you using?**
Renovate Pro

**Describe the bug**
This repo https://github.com/dmx-io/koa-joi-router has the following `renovate.json`:
```
{
  "commitMessagePrefix": "[renovate] ",
  "groupName": "all dependencies",
  "labels": [
    "renovate"
  ],
  "npmToken": "<redacted>",
  "packageRules": [
    {
      "packageNames": [
        "supertest"
      ],
      "allowedVersions": "<1.0.0"
    },
    {
      "packagePatterns": [
        "^@dmx/"
      ],
      "schedule": "at any time"
    }
  ],
  "schedule": "before 3am on the first day of the month"
}
```
but no PR was opened since the begin of the March until now.
At least `orbs.dmx_base` should be updated to `dmx-io/base@1.10.123` in `.circleci/config.yml`.

**Did you see anything helpful in debug logs?**
There's absolutely nothing for this repo in the logs:
<img width="1429" alt="screen shot 2019-03-05 at 01 17 59" src="https://user-images.githubusercontent.com/20856232/53771896-98e32780-3ee4-11e9-9e2e-c927119e4d5a.png">

**Additional context**

These repos didn't detect the schedule (each one has `Package not scheduled` in its logs):
- `dmx-io/dmx-api-gateway`
- `dmx-io/dmx-grades`
- `dmx-io/dmx-scheduling`
- `dmx-io/dmx-slackbot`
- `dmx-io/npm-data-models`

logs for `dmx-io/dmx-api-gateway`:
```
DEBUG: Compiling branchName: renovate/docker-all-dependencies(branch="renovate/docker-all-dependencies")
DEBUG: commitMessage: "[renovate] Update dmx_base orb to v1.10.123"(branch="renovate/docker-all-dependencies")
DEBUG: prTitle: "[renovate] Update dmx_base orb to v1.10.123"(branch="renovate/docker-all-dependencies")
DEBUG: generateBranchConfig(2)(branch="renovate/all-dependencies")

...

DEBUG: Compiling branchName: renovate/all-dependencies(branch="renovate/all-dependencies")
DEBUG: commitMessage: "[renovate] Update all dependencies"(branch="renovate/all-dependencies")
DEBUG: prTitle: "[renovate] Update all dependencies"(branch="renovate/all-dependencies")
DEBUG: config.repoIsOnboarded=true
INFO: Processing 2 branches: renovate/all-dependencies, renovate/docker-all-dependencies
DEBUG: Retrieving PR list
DEBUG: Retrieved 199 Pull Requests
DEBUG: PR hourly limit remaining: 5
DEBUG: processBranch with 2 upgrades(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: Retrieving branchList(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: branchExists(renovate/all-dependencies)=false(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: getBranchPr(renovate/all-dependencies)(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: findPr(renovate/all-dependencies, null, open)(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: branchExists=false(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: Branch has 2 upgrade(s)(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: recreateClosed is true(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: Checking schedule(before 3am on the first day of the month, null)(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
DEBUG: Checking 1 schedule(s)(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")

...

DEBUG: Package not scheduled(dependencies=["eslint","sinon"], branch="renovate/all-dependencies")
```

These repos have absolutely nothing in their logs:
- `dmx-io/dmx-admin-2`
- `dmx-io/koa-joi-router`
- `dmx-io/npm-dmx-sdk`
- `dmx-io/npm-grade-it-tools`
- `dmx-io/npm-koa-qs`
- `dmx-io/validation-schemas`

**Comment by rarkins** [comment]
@piotr-s-brainhub FYI you are using the Renovate App, not Renovate Pro. Renovate Pro is the self-hosted product.

**Comment by rarkins** [comment]
I checked the logs and found that on 24th February, GitHub returned 404 Not Found for some of your repositories, so they were marked as disabled and not checked again since. I've removed the disabled status from all your repos manually so that they'll get tried in the next hour. I've also updated Renovate's logic to set status to "unknown" rather than "disabled" in such cases so that they get retried the next hour, so it shouldn't happen again.

I have no idea why GitHub returned 404's. It only happened to the dmx-io account and only to some repositories and not all. I'll keep an eye out for this problem reoccurring but the above change should mean it doesn't matter too much if it happens again.

In the meantime, please either use the masterIssue to manually trigger PRs or let me know if you'd like me to manually force a null schedule on all your repos today so that you get PRs raised.

**Comment by piotr-s-brainhub** [comment]
@rarkins 

thanks, 404 for `dmx-io` was because we had some payments problems so GitHub denied access for some repos to all the users but later we paid so GitHub granted access again

**Comment by piotr-s-brainhub** [comment]
@rarkins 

renovate opened a PR for:
- `dmx-io/koa-joi-router`

however still I see no opened PR for:
- `dmx-io/dmx-api-gateway`
- `dmx-io/dmx-grades`
- `dmx-io/dmx-scheduling`
- `dmx-io/dmx-slackbot`
- `dmx-io/npm-data-models`
- `dmx-io/dmx-admin-2`
- `dmx-io/npm-dmx-sdk`
- `dmx-io/npm-grade-it-tools`
- `dmx-io/npm-koa-qs`
- `dmx-io/validation-schemas`
though the checking is hourly

**Comment by rarkins** [comment]
Yes that was the one I did manually but then I realised I should ask you first before doing more. I’ll queue them up later today. 

Thanks for the explanation re: the 404 - it was quite mysterious to me!

**Comment by rarkins** [comment]
@piotr-s-brainhub while checking the logs I just noticed that the `dmx-io` marketplace plan has been deactivated by GitHub since 24th Feb too, and not reactivated yet. Can you make sure marketplace billing is re-enabled and ask GitHub to reflect the same in their marketplace API?

**Comment by rarkins** [comment]
Perhaps you or whoever is the admin/owner can go to https://github.com/marketplace/renovate and see if you can just reactivate it that way

**Comment by piotr-s-brainhub** [comment]
@rarkins 

I've just installed it but still I have both `install` and `uninstall` buttons:

<img width="1064" alt="screen shot 2019-03-05 at 22 06 38" src="https://user-images.githubusercontent.com/20856232/53837586-2a10d780-3f93-11e9-8ef7-02d8c5e5bf7f.png">

**Comment by piotr-s-brainhub** [comment]
@rarkins 

I don't if it's related to this (maybe I should open a new issue):
- `dmx-io/dmx-admin-2` has the following `renovate.json`:
```
{
  "commitMessagePrefix": "[renovate] ",
  "groupName": "all dependencies",
  "labels": [
    "renovate"
  ],
  "npmToken": "<redacted>",
  "packageRules": [
    {
      "packagePatterns": [
        "^@dmx/"
      ],
      "schedule": "at any time"
    }
  ],
  "rangeStrategy": "bump",
  "schedule": "before 3am on the first day of the month"
```
- so beside the `@dmx/` packages, the update should be only once per month
- this PR https://github.com/dmx-io/dmx-admin-2/pull/231 with updating non-major packages was opened and merged
- among others it contains updating `react` from `^16.8.2` to `^16.8.3` and `react-dom` from `^16.8.2` to `^16.8.3`
- a new PR was opened for the same packages (`react` from `^16.8.3` to `^16.8.4` and `react-dom` from `^16.8.3` to `^16.8.4`)
- so this is unexpected as I was expecting next `react` and `react-dom` update on April 1

**Comment by rarkins** [comment]
@piotr-s-brainhub installing and uninstalling are a separate concept to marketplace. The app can be installed but with no marketplace plan. You need to go specifically to https://github.com/marketplace/renovate and click through the plans there

**Comment by rarkins** [comment]
@piotr-s-brainhub the reason you saw those PRs is that I force removed all schedules from your account temporarily. It's now back to your regular settings, although the bot is going to keep skipping over your repos until there's a marketplace plan in place again.

**Comment by rarkins** [comment]
@piotr-s-brainhub FYI, GitHub still shows your marketplace account as cancelled on 24th Feb. Can you take a screenshot of the current status and email it to me so that I can submit a ticket to their support? FYI I have left a manual override in place so that your account remains active in the App

**Comment by piotr-s-brainhub** [comment]
@rarkins 

thanks. I sent an email to support@renovatebot.com .
